src/main/java/com/fabriccommunity/spookytime/registry/SpookyEntities.java

```diff
@@ -23,6 +23,22 @@
 	public static ComponentType<CandyComponent> CANDY = ComponentRegistry.INSTANCE.registerIfAbsent(SpookyTime.id("candy"), CandyComponent.class);
 	
 	public static EntityType<PumpcownEntity> PUMPCOWN;
+
+	public static final EntityType<SpookyTreasureChestEntity> SPOOKY_TREASURE_CHEST = register(
+		"spooky_treasure_chest",
+		FabricEntityTypeBuilder.<SpookyTreasureChestEntity>create(
+			EntityCategory.MISC,
+			SpookyTreasureChestEntity::new
+		).size(EntityDimensions.fixed(.5f, .5f)).build()
+	);
+
+	public static final BlockEntityType<SpookyTreasureChestBlockEntity> SPOOKY_TREASURE_CHEST_BE = register(
+		"spooky_treasure_chest",
+		BlockEntityType.Builder.create(
+			SpookyTreasureChestBlockEntity::new,
+			SpookyBlocks.SPOOKY_TREASURE_CHEST
+		).build(null)
+	);
```

Comment:
This should be in `SpookyBlockEntities` (which also has a helper for registering BETypes).